usercommandmgmt/usercommandmgmt.py

```diff
@@ -0,0 +1,226 @@
+from redbot.core import Config, checks, commands, data_manager
+from .configurable import *
+from .commanddatahandler import *
+import inspect
+import sys
+from .commanddatahandler import Database
+import os
+
+
+# imports customcomm class
+path = os.path.join(
+    os.path.dirname(os.path.dirname(os.path.dirname(inspect.getfile(commands)))),
+    "cogs\customcom",
+)
+sys.path.insert(1, path)
+
+from customcom import (
+    CustomCommands,
+    CommandObj,
+    NotFound,
+    ArgParseError,
+    CommandNotEdited,
+)
```

Comment:
customcom is built into Red, use the actual import instead (also group this with the other `redbot` imports):
```suggestion
from redbot.cogs.customcom import CustomCommands
from redbot.cogs.customcom.customcom import (
    CommandObj,
    NotFound
    ArgParseError,
    CommandNotEdited,
)